ty.py

```diff
@@ -0,0 +1,6 @@
+from tqdm import tqdm 
+
```

Comment:
What is this file for?

Reply:
I have deleted it, I forgot to delete it earlier. My apologies